src/main/java/com/nutomic/syncthingandroid/syncthing/SyncthingService.java

```diff
@@ -291,6 +292,9 @@ private boolean isAllowedWifiConnected() {
     private void updateNotification() {
         SharedPreferences sp = PreferenceManager.getDefaultSharedPreferences(this);
         String type = sp.getString(PREF_NOTIFICATION_TYPE, "low_priority");
+        boolean foreground = sp.getBoolean(PREF_FOREGROUND_SERVICE, false);
+        if ("none".equals(type) && foreground)
+            type = "low_priority"; // foreground priority requires a notification
```

Comment:
No, because if the user selected "none", we would not even enter the "if" condition in line 299.

Above code just guarantees that _at least_ "min priority" is selected. I also wouldn't want my "default priority" notification to be downgraded to "min priority" just because I enable foreground service.